packages/backend/src/apps/toolbox/common/condition-is-true.ts

```diff
@@ -1,4 +1,29 @@
-import type { IJSONObject } from '@plumber/types'
+import type { IJSONObject, IJSONValue } from '@plumber/types'
+
+function compareNumbers(
+  field: IJSONValue,
+  condition: IJSONValue,
+  value: IJSONValue,
+): boolean {
+  // WARNING: can only compare safely up till Number.MAX_SAFE_INTEGER but BigInt cannot compare floats...
```

Comment:
we can use `big.js` if you are worried about this, but out of scope for this PR

Reply:
discussed to leave this out until user feedbacks